cocos/2d/CCDrawNode.h

```diff
@@ -313,12 +313,22 @@ class CC_DLL DrawNode : public Node
     
     // Overrides
     virtual void draw(Renderer *renderer, const Mat4 &transform, uint32_t flags) override;
+
+    void visit(Renderer* renderer, const Mat4 &parentTransform, uint32_t parentFlags) override;
```

Comment:
You can not just override virtual functions without using a keyword `virtual`. This is incorrect from the point of view of the original architecture.

Reply:
my fault, thx